### ISSUE TITLE
`comma-style`: does not report commas between default imports and named imports

### ISSUE THREAD

**Issue by ota-meshi** [eslint-stylistic/eslint-stylistic#599]
### Validations

- [X] Follow our [Code of Conduct](https://github.com/eslint-stylistic/eslint-stylistic/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guide](https://eslint.style/contribute/guide).
- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.

### Describe the bug

Does not report commas between default imports and named imports.

The following JavaScript code does not report any commas:

```js
/* eslint "@stylistic/comma-style": ["error", "last", { 
  "exceptions": {
    "ImportDeclaration": false,
  }
}] */
import a
  , {Foo} from 'module'
import b
  , {} from 'module'
import c,
  {Bar} from 'module'
import d,
  {} from 'module'
```

### Reproduction

https://stackblitz.com/edit/github-6uidhe-gy7x3z?file=src%2Findex.ts

### Contributes

- [X] I am willing to submit a PR to fix this issue
- [X] I am willing to submit a PR with failing tests
